cohere/generation.py

```diff
@@ -1,3 +1,5 @@
+import sseclient
```

Comment:
shouldn't this be in setup/requirements? or only imported when needed and documented as optional dependency